api/types/constants.go

```diff
@@ -127,6 +127,9 @@ const (
 	// KindNode is node resource
 	KindNode = "node"
 
+	// KindOpenSSHNode is a Teleport node.
+	KindTeleportNode = "teleport"
+
 	// KindOpenSSHNode is a registered OpenSSH (agentless) node.
 	KindOpenSSHNode = "openssh"
```

Comment:
Should these two maybe be renamed to `SubKindTeleportNode` and `SubKindOpenSSHNode`?

Reply:
Done